device-base/generate-dockerfile.sh

```diff
@@ -21,21 +35,34 @@ for device in $devices; do
 	;;
 	esac
 
-	for suite in $suites; do		
-
+	for suite in $suites; do
 		dockerfilePath="$device"
 		mkdir -p $dockerfilePath/$suite
-		sed -e s~#{FROM}~resin/$baseImage:$suite~g \
-			-e s~#{SUITE}~$suite~g $template > $dockerfilePath/$suite/Dockerfile
 
-	done
+		if [ $device == 'beaglebone' ]; then
 
-	# Only for armv7hf devices
-	if [ $device == 'raspberrypi2' ] || [ $device == 'beaglebone' ]; then
-		mkdir -p $dockerfilePath/sid
-		sed -e s~#{FROM}~resin/$baseImage:sid~g \
-			-e s~#{SUITE}~sid~g $template > $dockerfilePath/sid/Dockerfile		
-	fi
+			case "$suite" in
+			'wheezy')
+				sourcelist="$bb_sourceslist_wheezy_cmd"
```

Comment:
What did you try? You would have to replace the `$sourcelist` uses with `${!sourcelist}` - it's not a big deal though as I'm not sure dropping the switch to use that style would make the code any clearer anyway